tests/run-make/pdb-buildinfo-cl-cmd/rmake.rs

```diff
@@ -0,0 +1,31 @@
+// Check if the pdb file contains the following information in the LF_BUILDINFO:
+// 1. full path to the compiler (cl)
+// 2. the commandline args to compile it (cmd)
+// This is because these used to be missing in #96475.
+// See https://github.com/rust-lang/rust/pull/113492
+
+//@ only-windows-msvc
+// Reason: pdb files are unique to this architecture
+
+use run_make_support::{assert_contains, bstr, env_var, rfs, rustc};
+
+fn main() {
+    rustc()
+        .input("main.rs")
+        .arg("-g")
+        .crate_name("my_crate_name")
+        .crate_type("bin")
+        .metadata("dc9ef878b0a48666")
+        .run();
+    use bstr::ByteSlice;
+    assert!(&rfs::read("my_crate_name.pdb").find(env_var("RUSTC").as_bytes()).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""main.rs""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""-g""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""--crate-name""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""my_crate_name""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""--crate-type""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""bin""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""-C""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""metadata=dc9ef878b0a48666""#).is_some());
+    assert!(&rfs::read("my_crate_name.pdb").find(br#""--out-dir""#).is_some());
```

Comment:
Suggestion: a helper like `assert_pdb_contains`, probably just in this test file.